### PR TITLE
Pass content/mime type for files, or default based on extension

### PR DESCRIPTION
**Pull request by KBalderson**

### CODE REVIEW

**Comment by timfjord** [comment]
@KBalderson thanks for the contribution. But can you please add specs for `Tempfile#content_type`

**Comment by KBalderson** [comment]
@timsly sorry for the long delay, I've added the specs for content_type.

**Comment by KBalderson** [comment]
After #7 is merged, I'll rebase and push to gets the 2.3 specs to pass

**Comment by timfjord** [comment]
@KBalderson sounds great. Can you please also squash commits
By the way, #7 is already merged

File: spec/tempfile_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    subject(:content_type) { tempfile.content_type }
+
+    describe 'without a content_type supplied' do
+      specify { expect(content_type).to eq 'image/gif' }
+    end
+
+    describe 'with a specified content_type' do
+      let(:options) { { content_type: 'image/jpeg' } }
+      specify { expect(content_type).to eq 'image/jpeg' }
+
```
Comment (timfjord) [inline spec/tempfile_spec.rb]: @KBalderson can you pleases remove this extra space here

**Comment by KBalderson** [comment]
white space removed, commit squashed.

**Comment by timfjord** [comment]
Great. Thanks :+1:

**Pull request merged**